muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/sass/elements/link.scss

```diff
@@ -267,6 +267,7 @@
     background-color: $color-main-function-label-link-selected-background;
     
     .text {
```

Comment:
I know we're getting rid of these .text classes, but - in general - this override I don't like. I'm not sure it's a good practice to override some other components classes. I mean this can be done in a number of ways; Why like this?

Reply:
Only other way to achieve this currently is to add another class name to markup like text--selected-label when either  communicator or guider label link is selected inside label drop down box. That would add in my opinion unnecessary overhead as we need to add corresponding class name to markup and as well add extra css rules to match that class name instead of just override some css rules based on selected state of parent element.

How ever this changes completely when we get rid of text class name.